controller_manager/test/test_spawner_unspawner.cpp

```diff
@@ -811,8 +811,9 @@ TEST_F(TestLoadController, test_spawner_parsed_controller_ros_args)
 
   // Now test the remapping of the service name with the controller_ros_args
   EXPECT_EQ(
-    call_spawner(
-      "ctrl_2 -c test_controller_manager --controller-ros-args '-r /ctrl_2/set_bool:=/set_bool'"),
+    call_spawner("ctrl_2 -c test_controller_manager --controller-ros-args '-r "
+                 "/ctrl_2/set_bool:=/set_bool -p run_cycle:=20' --controller-ros-args '-r "
+                 "/ctrl_1/set_value:=/set_value'"),
```

Comment:
you want to check for the /set_value as well?

Reply:
Not really. There is nothing related. This is just to test that it doesn't break at spawner level

Reply:
I can move the parameter part to the second part. May be that's more robust way

Reply:
I see 3 --ros-args, two remap rules and one parameter. Why just don't check for all the three of them?

Reply:
Because the second remapping doesn't exist. I just added casually. May be I can remove it

Reply:
ok sorry, I haven't checked that carefully. If we want to support space-separated rules with a single --controller-ros-args, we should somewhen also test it.